packages/neos-ui/src/Containers/Modals/NodeCreationDialog/index.js

```diff
@@ -50,51 +99,13 @@ export default class NodeCreationDialog extends PureComponent {
         secondaryInspectorComponent: null
     };
 
-    state = NodeCreationDialog.getDerivedStateFromProps(
-        this.props,
-        NodeCreationDialog.defaultState
-    );
```

Comment:
This doesnt work anymore with esbuild ^^ - so we extracted the helpers to the module scope ;)